### PR TITLE
[#3921] springdoc swagger

### PR DESCRIPTION
**Pull request by jsokolowskii**
Ref: https://github.com/Activiti/Activiti/issues/3921

### CODE REVIEW

**Comment by jsokolowskii** [comment]
This PR should wait for Reading Release Train.

File: activiti-cloud-audit-service/activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/swagger/AuditSwaggerIT.java [hunk 1]
```diff
@@ -26,13 +26,9 @@
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.web.servlet.MockMvc;
 
-import static org.hamcrest.Matchers.equalTo;
-import static org.hamcrest.Matchers.hasKey;
-import static org.hamcrest.Matchers.startsWith;
+import static org.hamcrest.Matchers.*;
```
Comment (VitoAlbano) [inline activiti-cloud-audit-service/activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/swagger/AuditSwaggerIT.java]: please avoid starred imports

File: activiti-cloud-audit-service/activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/swagger/AuditSwaggerIT.java [hunk 2]
```diff
@@ -26,13 +26,9 @@
...
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
```
Comment (VitoAlbano) [inline activiti-cloud-audit-service/activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/swagger/AuditSwaggerIT.java]: as above

File: activiti-cloud-query-service/activiti-cloud-starter-query/src/test/java/org/activiti/cloud/starter/tests/swagger/QuerySwaggerIT.java [hunk 3]
```diff
@@ -40,6 +27,11 @@
 import org.springframework.test.web.servlet.MockMvc;
 import org.springframework.test.web.servlet.MvcResult;
 
+import static net.javacrumbs.jsonunit.assertj.JsonAssertions.assertThatJson;
+import static org.hamcrest.Matchers.*;
```
Comment (VitoAlbano) [inline activiti-cloud-query-service/activiti-cloud-starter-query/src/test/java/org/activiti/cloud/starter/tests/swagger/QuerySwaggerIT.java]: as above

**Review by VitoAlbano — CHANGES_REQUESTED** [review]

File: activiti-cloud-audit-service/activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/swagger/AuditSwaggerIT.java (hunk 1 above)
Comment (jsokolowskii) [reply activiti-cloud-audit-service/activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/swagger/AuditSwaggerIT.java]: done

File: activiti-cloud-audit-service/activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/swagger/AuditSwaggerIT.java (hunk 2 above)
Comment (jsokolowskii) [reply activiti-cloud-audit-service/activiti-cloud-starter-audit/src/test/java/org/activiti/cloud/starter/audit/tests/it/swagger/AuditSwaggerIT.java]: done

File: activiti-cloud-query-service/activiti-cloud-starter-query/src/test/java/org/activiti/cloud/starter/tests/swagger/QuerySwaggerIT.java (hunk 3 above)
Comment (jsokolowskii) [reply activiti-cloud-query-service/activiti-cloud-starter-query/src/test/java/org/activiti/cloud/starter/tests/swagger/QuerySwaggerIT.java]: done

**Review by VitoAlbano — APPROVED** [review]

**Pull request merged**